apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/SearchResultsContainer.tsx

```diff
@@ -1,31 +1,40 @@
-import { useTranslation } from 'next-i18next';
+import { useSearchTranslation } from 'events-helsinki-components';
 import React from 'react';
 import { ContentContainer, PageSection } from 'react-helsinki-headless-cms';
-
+import OrderBySelect from './OrderBySelect';
 import ResultsInfoContainer from './ResultsInfo';
 import styles from './searchResultList.module.scss';
 
 interface Props {
   loading: boolean;
   eventsCount: number;
   eventList: React.ReactElement;
+  showOrderBySelect?: boolean;
 }
 
 const SearchResultsContainer: React.FC<Props> = ({
   loading,
   eventsCount,
   eventList,
+  showOrderBySelect,
```

Comment:
this could have a default. `showOrderBySelect=false`

Reply:
I didn't add it because of the type being `showOrderBySelect?: boolean;` so just leaving it undefined makes it falsy. Is there a common tradition in Javascript/Typescript related to giving default values or is leaving props undefined and letting them be evaluated to falsy in boolean contexts as good? Looking at this from Python point of view Zen of Python could be used to argue in favor of adding the explicit default value because of "explicit is better than implicit" but as this is not Python but Javascript/Typescript that doesn't apply here. What are the traditions/norms for Javascript/Typescript code or are they defined anywhere—like in Zen of Python?

Reply:
Chose to add this because it made sense to me after removing PyCharm's inline hints from view. Without the PyCharm's inline hints adding the default value makes the code clearer in my opinion, with the inline hints there was more justification IMO for not adding the default value. As the inline hints are not part of the code but a feature an IDE gives they can't be counted upon by everyone.

With the inline hints:
![image](https://user-images.githubusercontent.com/77663720/213106083-0e94f9f1-e4df-4ad4-b54d-5ec98f8ab28c.png)

Without the inline hints:
![image](https://user-images.githubusercontent.com/77663720/213106161-f6178362-c0fb-4aaf-a658-c69527ddc16e.png)